test/docfx.Test/TestUtility.cs

```diff
@@ -131,6 +131,44 @@ public static IDisposable EnsureFilesNotChanged(string path, bool skipInputCheck
             }
         }
 
+        public static void SendLanguageServerNotification(
```

Comment:
nit: consider move this to `DocfxLanguageServerTestHost.cs` as it is lang server explicit. If there is a need to reuse `ApplyVariables`, we could extract that later.